contract/src/start-game1-proposal.js

```diff
@@ -1,7 +1,8 @@
 // @ts-check
-import { E } from '@endo/far';
+import { E } from '@endo/eventual-send';
```

Comment:
`@endo/far` is the norm / convention we want to teach. Please revert.
**important**